### DISCUSSION TITLE
did anyone know how to downgraded chrome version?

### DISCUSSION THREAD

**Discussion by hank929** [3kh0/ext-remover#970] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
did anyone know how to downgraded chrome version?

**Comment by AshtonDavies** [comment]
Patched, but I heard there's a workaround in Titanium Network. The old, regular method should be in the README in this repository.

**Reply by hank929** [reply]
let me try

**Reply by hank929** [reply]
IDK how to use that......

**Reply by nathblade16** [reply, marked as answer]
here is my downgrading method hope it helps
downgrading is unpatched but up to a certain point depending on your Chromebooks kernel version
how to check kernel version:
1. press escape refresh and power
2. press tab
3. find where it says something like kernel version
4. the last number is your kernel version
if it is 0 or 1 you can downgrade to any version
if it is 2 you can downgrade between 112 and 119
if it is 3 you can downgrade between 120 and latest
source a recovery image for the version that you want and follow normal downgrading steps

**Reply by v4pl** [reply]
Let him join the discord group @AshtonDavies

**Reply by hank929** [reply]
ty

**Reply by hank929** [reply]
how to join?
on discord?

**Reply by regularmovie** [reply]
@bloohacked how do you source a recovery image

**Comment by nathblade16** [comment]
> @bloohacked how do you source a recovery image

I get them from here [source](https://chromiumdash.appspot.com/serving-builds?deviceCategory=ChromeOS)

**Reply by regularmovie** [reply]
i know im octupus but there is multiple options for octupus
